www/include/monitoring/objectDetails/template/serviceDetails.ihtml

```diff
@@ -651,3 +650,9 @@ var clipboard = new Clipboard('.btn-copy', {
 
 </script>
 {/literal}
+
+{literal}
+<script>
+    formatDateMoment();
+</script>
+{/literal}
```

Comment:
It is not possible to call the function directly after the bracket? This will avoid closing the "script" and "literal" tags to open them again afterwards.

Reply:
change done